projects/packages/search/src/customizer/customize-controls/class-label-control.php

```diff
@@ -8,10 +8,15 @@
 
 namespace Automattic\Jetpack\Search;
 
+use WP_Customize_Control;
+
+if ( ! class_exists( 'WP_Customize_Control' ) ) {
+	return;
```

Comment:
In this case, would it be better to do the necessary `require_once ABSPATH . WPINC . '/class-wp-customize-control.php';` instead? We can assume WordPress is available, unlike for the WP_CLI issue previously where we probably can't.